### PR TITLE
Cleanup build scripts

### PR DESCRIPTION
**Pull request by tamird**
Remove duplicate files throughout

### CODE REVIEW

File: src/ProtocolBuffers/runtime-pb-swift/Descriptor.pb.swift [hunk 1]
```diff
@@ -68,6 +68,7 @@ public func == (lhs: PBDescriptorProto, rhs: PBDescriptorProto) -> Bool {
   fieldCheck = fieldCheck && (lhs.extensionRange == rhs.extensionRange)
   fieldCheck = fieldCheck && (lhs.extension_ == rhs.extension_)
   fieldCheck = fieldCheck && (lhs.hasOptions == rhs.hasOptions) && (!lhs.hasOptions || lhs.options == rhs.options)
+  fieldCheck = fieldCheck && (lhs.oneofDecl == rhs.oneofDecl)
```
Comment (tamird) [inline src/ProtocolBuffers/runtime-pb-swift/Descriptor.pb.swift]: this is just the result of recompiling the descriptors

**Comment by alexeyxo** [comment]
Descriptor.proto:
```protobuf
message DescriptorProto {
  optional string name = 1;

  repeated FieldDescriptorProto field = 2;
  repeated FieldDescriptorProto extension = 6;

  repeated DescriptorProto nested_type = 3;
  repeated EnumDescriptorProto enum_type = 4;

  message ExtensionRange {
    optional int32 start = 1;
    optional int32 end = 2;
  }
  repeated ExtensionRange extension_range = 5;

  optional MessageOptions options = 7;
}
}
```
You have a bug! Where did "oneofDecl"?

**Comment by tamird** [comment]
I updated descriptor.proto -- oneOf is a new feature in 2.6.

**Comment by alexeyxo** [comment]
Strange ... I updated it... Ok.

**Comment by alexeyxo** [comment]
Thanks a lot!

**Pull request merged**